tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/ssl/CoreClientOverOneWaySSLTest.java

```diff
@@ -85,9 +85,9 @@ public static Collection getParameters() {
          {"SunJCE", "JCEKS", false, false},
          {"SUN", "JKS", false, false},
          {"SunJSSE", "PKCS12", false, false},
-         {"JCEKS", null, true, false}, // for compatibility with old keyStoreProvider
-         {"JKS", null, true, false},   // for compatibility with old keyStoreProvider
-         {"PKCS12", null, true, false} // for compatibility with old keyStoreProvider
+         {"JCEKS", null, false, false}, // for compatibility with old keyStoreProvider
+         {"JKS", null, false, false},   // for compatibility with old keyStoreProvider
+         {"PKCS12", null, false, false} // for compatibility with old keyStoreProvider
```

Comment:
I don't understand why these changes were necessary. This eliminates the tests for the warning log message.

Reply:
To pass tests for this PR - otherwise quorum tests are not running. Expected changes in first commit only

Reply:
this test started to fail because Zookeeper 3.8.x uses logback. rollback to 3.7.1

Reply:
I'm not sure that Zookeeper will work after such changes

Reply:
I took a look at the ZooKeeper code. As far as I can tell it's using SLF4J as the API (just like the broker), and Logback is the implementation. I think you'd be fine to exclude Logback via the `pom.xml`.

Reply:
done